ci/build-tarballs.sh

```diff
@@ -13,8 +13,8 @@
 set -ex
 
 platform=$1
-src=$2
-exe=$3
+src=$1
+exe=$2
```

Comment:
???

Reply:
This is an internal script we use to build tarballs, it's not really intended to be super-well-documented or make a ton of sense...

Reply:
Just for future sanity -- maybe standardize on `$platform`, and add a `# Usage: build-tarballs.sh PLATFORM [.exe] (PLATFORM is e.g. x86_64-linux, aarch64-linux)` comment (no need for an actual arg check)?